spring-boot-samples/pom.xml

```diff
@@ -68,6 +68,7 @@
 		<module>spring-boot-sample-tomcat7-jsp</module>
 		<module>spring-boot-sample-traditional</module>
 		<module>spring-boot-sample-undertow</module>
+		<module>spring-boot-sample-undertow-access-log</module>
```

Comment:
Rather than creating a whole new sample, I think that access log configuration could just be added to `spring-boot-sample-undertow`